lock/src/main/java/com/auth0/android/lock/views/ClassicPanelHolder.java

```diff
@@ -320,6 +348,8 @@ public void onClick(View v) {
             event = changePwdForm.submitForm();
         } else if (customFieldsForm != null) {
             event = customFieldsForm.submitForm();
+        } else if (mfaCodeFormView != null) {
```

Comment:
Now seeing all these methods with the `if not nil then do something` for each form I think we need to handle this better since it doesn't scale and is highly error prone